apps/finance/contracts/Finance.sol

```diff
@@ -258,11 +256,36 @@ contract Finance is EtherTokenConstant, IsContract, AragonApp {
         payment.maxRepeats = _maxRepeats;
         payment.createdBy = msg.sender;
 
-        if (_nextPaymentTime(paymentId) <= getTimestamp64()) {
-            _executePayment(paymentId);
-        }
+        // We skip checking how many times the new payment was executed to allow creating new
+        // recurring payments before having enough vault balance
+        _executePayment(paymentId);
     }
 
+    /**
+    * @notice Create a new payment of `@tokenAmount(_token, _amount)` to `_receiver` for '`_reference`'
+    * @param _token Address of token for payment
+    * @param _receiver Address that will receive payment
+    * @param _amount Tokens that are paid every time the payment is due
+    * @param _reference String detailing payment reason
+    */
+   function newPaymentTransaction(address _token, address _receiver, uint256 _amount, string _reference)
+        external
+        authP(CREATE_PAYMENTS_ROLE, arr(_token, _receiver, _amount, MAX_UINT, 1))
```

Comment:
I think most users will parametrize it with a `>`, as setting a payment with a very low interval can drain the vault.

Reply:
Yes, that makes sense, but actually both could be set using an `AND` operator, and then this function would not work.

Reply:
I chose `MAX_UINT64` against `0` since most people will probably want `>`.

I'll document it, but another thing we could do is use `MAX_UINT256` instead here, as a sentinel interval value for "never".

Reply:
That makes sense